cmd/info.go

```diff
@@ -108,11 +116,12 @@ func info(ctx *cli.Context) error {
 			continue
 		}
 
-		wb := utils.NewBuffer(8 + 10)
+		wb := utils.NewBuffer(8 + 11)
 		wb.Put32(meta.InfoV2)
-		wb.Put32(10)
+		wb.Put32(11)
 		wb.Put64(inode)
 		wb.Put8(recursive)
+		wb.Put8(fast)
```

Comment:
client in old version will use it as raw

Reply:
Fixed